aws/services/amazonmq/resource.ftl

```diff
@@ -6,7 +6,17 @@
             "UseRef" : true
         },
         DNS_ATTRIBUTE_TYPE : {
-            "Attribute" : "AmqpEndpoints"
+            "Value" : {
+                "Fn::Join" : [
+                    ";",
+                    {
+                        "Fn::GetAtt" : [
+                            "mqBrokerXmgmtXrabbithutch",
```

Comment:
No other reference to this in the PR. I was not expecting anything in the master data for this ?